src/api_midlevel.jl

```diff
@@ -46,6 +46,83 @@ function get_extent_dims(obj::Union{Dataspace,Dataset,Attribute})
     return dims, maxdims
 end
 
+
+"""
+    HDF5.get_chunk_offset(dataset_id, index)
+
+Get 0-based offset of chunk from 0-based index.
+
+For a 1-based API, see HDF5.ChunkStorage.
+
+The returned offsets are in Julian order than HDF5's C-order.
+"""
+function get_chunk_offset(dataset_id, index)

```

Comment:
Is this and the following method needed somewhere?

Reply:
It would be good to test these methods

Reply:
At the moment `get_chunk_offset` is used by the `h5d_read_chunk` and `h5d_write_chunk`.  I don't think `get_chunk_index` is currently used or tested. Let me see what I can do about that.